### PR TITLE
rpm: split development files into isolated package

### PR DESCRIPTION
**Pull request by kenhys**
It fixes: W: devel-file-in-non-devel-package

### CODE REVIEW

File: td-agent/yum/td-agent.spec.in [hunk 1]
```diff
@@ -71,6 +71,13 @@ Requires(pre): /usr/bin/getent, /usr/sbin/adduser
 %description
 The stable distribution of Fluentd, called td-agent.
 
+%package devel
+Summary:  Development files for td-agent
+Requires: %{name} = %{version}-%{release}
+
+%description devel
+The stable distribution of Fluentd development package, called td-agent-devel.
```
Comment (ashie) [inline td-agent/yum/td-agent.spec.in:79]: `, called td-agent-devel` should be removed as you do it in #78

File: td-agent/yum/td-agent.spec.in [hunk 2]
```diff
@@ -101,7 +108,14 @@ fi
...
 %files
 %defattr(-,root,root,-)
-/opt/*
+/opt/td-agent/LICENSES/*
+/opt/td-agent/bin/*
+/opt/td-agent/lib/ruby/*
+/opt/td-agent/share/*
+/opt/td-agent/lib/libruby.so*
+/opt/td-agent/lib/libjemalloc.so*
+/opt/td-agent/lib/libjemalloc*.a
```
Comment (ashie) [inline td-agent/yum/td-agent.spec.in:117]: libjemalloc*.a isn't used, we can drop it.
(It should be moved to `td-agent-static` package if we keep the file.)

**Comment by ashie** [comment]
I'm not sure whether we should split development files into an isolated package or not.
We should check the impact on existing users before doing it.

File: td-agent/yum/td-agent.spec.in (hunk 1 above)
Comment (kenhys) [reply td-agent/yum/td-agent.spec.in:79]: Ah, it should be fixed.

File: td-agent/yum/td-agent.spec.in (hunk 2 above)
Comment (kenhys) [reply td-agent/yum/td-agent.spec.in:117]: It will be removed by #76.

**Comment by kenhys** [comment]
It is good practice to split devel, but for td-agent, split development files into another package is not so
good because gem install may fail if -devel package is not installed with native gem.

**Comment by kenhys** [comment]
Changed to suppress W: devel-file-in-non-devel-package.

**Pull request closed without merging**